linux_pipeline/Jenkinsfile_developer_comparison_validation

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env groovy
```

Comment:
what happens if first job fails after 5 seconds and the second one takes 1:30h? will the job exit after 5 sec or after 1:30h?